### ISSUE TITLE
Add support to serverless 4

### ISSUE THREAD

**Issue by cernadasjuan** [claygregory/serverless-prune-plugin#65]
Right now if you try to use it will serverless 4, it shows the following error when installing:

```
npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: serverless-prune-plugin@2.0.2
npm ERR! Found: serverless@4.0.23
npm ERR! node_modules/serverless
npm ERR!   dev serverless@"^4.0.23" from the root project
npm ERR!   peer serverless@">= 1.26.0" from serverless-plugin-canary-deployments@0.8.0
npm ERR!   node_modules/serverless-plugin-canary-deployments
npm ERR!     dev serverless-plugin-canary-deployments@"^0.8.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer serverless@"1 || 2 || 3" from serverless-prune-plugin@2.0.2
npm ERR! node_modules/serverless-prune-plugin
npm ERR!   dev serverless-prune-plugin@"^2.0.2" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: serverless@3.38.0
npm ERR! node_modules/serverless
npm ERR!   peer serverless@"1 || 2 || 3" from serverless-prune-plugin@2.0.2
npm ERR!   node_modules/serverless-prune-plugin
npm ERR!     dev serverless-prune-plugin@"^2.0.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by dgoerdes** [comment]
Pull Request is already open: https://github.com/claygregory/serverless-prune-plugin/pull/68

**Comment by claygregory** [comment]
Hi all — PR #68 has been merged and released as v 2.1.0
